elasticdl_client/README.md

```diff
@@ -0,0 +1,8 @@
+# ElasticDL Client
+
+This is the client command-line tool for ElasticDL. We can use it to build
```

Comment:
This is more straightforward:

`ElasticDL client is a command-line tool. Users can use it to prepare and submit ElasticDL jobs.`

Reply:
Good suggestion. Make some update based on this comment.